www/class/centreonUtils.class.php

```diff
@@ -90,6 +90,29 @@ public function visit($idPage, $redirect = true)
         }
     }
 
+    /**
+     * used to sanitize key and value in array
+     * @param array<mixed> $inputArray
+     * @return array<mixed>
+     */
+    public function sanitizeInputArrayNew(array $inputArray): array
+    {
+        $sanitizedArray = [];
+        foreach ($inputArray as $key => $value) {
+            if ($key === 'address') {
```

Comment:
The centreonUtils class must not contain any method containing specific code

Reply:
done